### PR TITLE
crud operation completed

### PR DESCRIPTION
**Pull request by rutulsheladiya31**

### CODE REVIEW

File: js/script.js [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+        swal("Oops", "Please Select Product Image", "error");
+        return false;
+    } else if (!allowImg.exec(productImage)) {
+        swal("Oops", "Please Enter Valid Image (.jpg,.png,.jpeg,.gif)", "error");
+        return false;
+    }
+
+    let data1=localStorage.getItem(document.getElementById("productid").value);
+    if (data1){
+        swal("Oops", "You Can not insert this data because this key is alredy exist.", "error");
```
Comment (kerulshahsimform) [inline js/script.js:167]: Already*

File: js/script.js [hunk 2]
```diff
@@ -0,0 +1,207 @@
...
+        return false;
+    }
+    return true;
+    
+}
+
+// function for validation at update time
+function updatevalidation() {
```
Comment (kerulshahsimform) [inline js/script.js:175]: Can we merge the validations for create and update?
As we can add conditions in which we need to apply validations in create or update mode OR on both.

File: js/script.js (hunk 1 above)
Comment (rutulsheladiya31) [reply js/script.js]: Resolved.

File: js/script.js (hunk 2 above)
Comment (rutulsheladiya31) [reply js/script.js]: Yes, those validation are common at the time of insert and update that i have put globally. and other validations i put based on insert & update conditions.

**Review by kerulshahsimform — APPROVED** [review]

**Review by kerulshahsimform — APPROVED** [review]

**Review by kerulshahsimform — APPROVED** [review]

**Pull request merged**